tests/dragonfly/connection_test.py

```diff
@@ -419,18 +420,68 @@ async def test_large_cmd(async_client: aioredis.Redis):
 
 
 @pytest.mark.asyncio
-async def test_reject_non_tls_connections_on_tls_master(with_tls_args, df_local_factory):
-    master = df_local_factory.create(admin_port=1111, port=1211, **with_tls_args)
+async def test_reject_non_tls_connections_on_tls_master(with_tls_server_args, df_local_factory):
+    master = df_local_factory.create(admin_port=1111, port=1211, **with_tls_server_args)
     master.start()
 
     # Try to connect on master without admin port. This should fail.
     client = aioredis.Redis(port=master.port)
     try:
         await client.execute_command("DBSIZE")
-        raise "Non tls connection connected on master with tls. This should NOT happen"
     except redis_conn_error:
         pass
 
     # Try to connect on master on admin port
     client = aioredis.Redis(port=master.admin_port)
     assert await client.ping()
+    await client.close()
+
+
+@pytest.mark.asyncio
+async def test_tls_insecure(with_ca_tls_server_args, with_tls_client_args, df_local_factory):
+    master = df_local_factory.create(port=BASE_PORT, **with_ca_tls_server_args)
```

Comment:
why do you need to specify `BASE_PORT` ?

Reply:
not a particular reason, it's a convention I seen throughout the tests and I thought I should follow it